.github/workflows/production.yml

```diff
@@ -84,6 +84,7 @@ jobs:
           GEMINI_API_KEY: ${{ secrets.GEMINI_API_KEY }}
           REQUESTS_PER_MINUTE: ${{ secrets.REQUESTS_PER_MINUTE }}
           LIMIT_WINDOW: ${{ secrets.LIMIT_WINDOW }}
+          ADMIN_USERS: ${{ secrets.ADMIN_USERS }}
```

Comment:
You need to add `echo ADMIN_USERS=${{ secretsADMIN_USERS }} >> .env` in the run section below as well.